api.rst

```diff
@@ -60,12 +60,23 @@ like          LIKE operator (use * in place of %)
 ilike         ILIKE operator (use * in place of %)
 in            one of a list of values e.g. :code:`?a=in.1,2,3` – also supports commas in quoted strings like :code:`?a=in."hi,there","yes,you"`
 is            checking for exact equality (null,true,false)
-@@            full-text search using to_tsquery
-@>            contains e.g. :code:`?tags=@>.{example, new}`
-<@            contained in e.g. :code:`?values=<@{1,2,3}`
+fts           full-text search using to_tsquery
```

Comment:
i think it would be useful to have a third column that shows the exact postgresql operator/symbol.
It's not immediately clear what "strictly left" would actually be.
other than that, great job on pr and this